back/app/policies/admin_publication_policy.rb

```diff
@@ -13,8 +13,22 @@ def resolve
 
     def scope_for_klass(klass)
       # If the publication is a project, we usually hide hidden projects
-      if klass == Project && !context[:include_hidden]
-        scope_for(klass).not_hidden
+      if klass == Project
+        scope = scope_for(klass)
+
+        if !context[:include_hidden]
+          scope = scope.not_hidden
+        end
+
+        if context[:apply_listed_scope]
+          scope = ProjectPolicy.apply_listed_scope(
+            scope,
+            user,
+            context[:include_unlisted] != false
+          )
+        end
```

Comment:
Think you can probably remove `:apply_listed_scope` from the context as I noticed it is always `true` by doing something like:
```suggestion
        if context.key?[: include_unlisted]
          scope = ProjectPolicy.apply_listed_scope(
            scope,
            user,
            context[:include_unlisted]
          )
        end
```

Reply:
I renamed the param to `remove_unlisted`,  and gave it three possible values: `nil` (do not remove unlisted), `'remove_all_unlisted'`, and `'remove_unlisted_that_user_cannot_moderate'`. This seemed to be the most logical, since in most cases we actually don't remove unlisted projects. So you have to specifically say that you want to remove unlisted projects, and then you need to specify if you want to remove all of them or just the ones that the user cannot moderate.